natgeo/mortar
# Feature/mortar 77

https://jira.nationalgeographic.com/browse/MORTAR-77

Testing instructions: 

- git checkout
- gulp serve
- visit http://localhost:3000/patterns and check to see if icon clipping bug still occurs
